src/components/ButtonPanel.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => {
+  const [buttons] = useState([
+    ['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', 'X'],
+    ['4', '5', '6', '-'],
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ]);
+
+  const [groupIndices] = useState([1, 2, 3, 4]);
+
+  return (
+    <div>
+      {buttons.map((group, outerIndex) => (
+        <div className="btn-group" key={groupIndices[outerIndex]}>
+          {group.map(character => (
+            <Button key={character} name={character} />
+          ))}
+        </div>
+      ))}
+    </div>
+  );
+};
+
+export default ButtonPanel;
```

Comment:
I know accesing the state through the react hook `useState` is cool and rendering the buttons using `.map` is efficient, but as required by the project each button should be hardcoded jsx.
![scr](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/buttonPanel/buttons.png)